### PR TITLE
Move footnotes to markdown basics

### PR DESCRIPTION
**Pull request by cwickham**
Closes https://github.com/quarto-dev/quarto-cli/issues/9964

Footnotes move to: [Guide > Authoring > Markdown Basics](https://deploy-preview-1188.quarto.org/docs/authoring/markdown-basics.html#footnotes)

Citations are now stand-alone: [Guide > Authoring > Scholarly Writing > Citations](https://deploy-preview-1188.quarto.org/docs/authoring/citations.html)

### CODE REVIEW

File: docs/authoring/citations.qmd [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+title: Citations
 format: html
 link-citations: true
-section-title-footnotes: Example Footnotes
+aliases: 
+  - /docs/authoring/footnotes-and-citations.html
 ---
 
-## Citations
+## Overview
```
Comment (cderv) [inline docs/authoring/citations.qmd:9]: Should we add an id to not break any existing link in the wild ? 

```suggestion
## Overview {#citations}
```

Or not that important and more messy than useful ? There is probably no link to it really, but this is probably something to remember when we reorganize docs IMO

**Review by cderv — APPROVED** [review]
LGTM ! Thanks !

**Comment by cderv** [comment]
@cwickham will you handle the port to prerelease doc ? Or do you want me to handle it ?

**Comment by cwickham** [comment]
@cderv I'll do it!

**Pull request merged**